src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -52,6 +52,21 @@
         <li><p>After you return to the Certificate screen, click <strong>OK</strong>.</p></li>
     </ol>
 
+    <stache-page-anchor>
+        Installation for Firefox
```

Comment:
Is this for Windows only?

Reply:
@mattfurness mentioned above he tested the instructions on macOS.